packages/groovy/build.sh

```diff
@@ -8,13 +8,17 @@ _JANSI_VERSION=2.4.1
 _JLINE_VERSION=2.14.6
```

Comment:
If we're doing it for all these other packages,
we may as well fold these into a `TERMUX_PKG_VERSION=(...)` array.
The `_JANSI_TAGNAME` and `_JLINE2_TAGNAME` abstractions are probably still useful to keep though.